bridge/Mixer.cpp

```diff
@@ -1390,29 +1343,25 @@ bool Mixer::reconfigureVideoStream(const std::string& endpointId,
     videoStream->secondarySimulcastStream = secondarySimulcastStream;
     std::memcpy(&videoStream->ssrcWhitelist, &ssrcWhitelist, sizeof(SsrcWhitelist));
 
-    EngineCommand::Command command;
-
     if (secondarySimulcastStream.isSet())
     {
-        command.type = EngineCommand::Type::ReconfigureVideoStreamSecondary;
-        command.command.reconfigureVideoStreamSecondary.mixer = &_engineMixer;
-        command.command.reconfigureVideoStreamSecondary.simulcastStream = videoStream->simulcastStream;
-        command.command.reconfigureVideoStreamSecondary.secondarySimulcastStream =
-            videoStream->secondarySimulcastStream.get();
-        command.command.reconfigureVideoStreamSecondary.transport = videoStream->transport.get();
-        std::memcpy(&command.command.reconfigureVideoStreamSecondary.ssrcWhitelist,
-            &ssrcWhitelist,
-            sizeof(SsrcWhitelist));
+        _engine.post(utils::bind(&EngineMixer::reconfigureVideoStream,
+            _engineMixer.get(),
+            videoStream->transport.get(),
```

Comment:
What if we have another reconfigure before the engine thread completes this. We will have data in videoStream that is being updated.

Reply:
Perhaps it would be better give a copy of `ssrcWhitelist` and simulcasts if that fits in `utils::function` storage